tensorboard/functionaltests/core_test.py

```diff
@@ -63,7 +63,7 @@ def tearDownClass(cls):
   def setUp(self):
     self.driver = webtest.new_webdriver_session()
     self.driver.get("http://localhost:%s" % self.port)
-    self.wait = wait.WebDriverWait(self.driver, 2)
+    self.wait = wait.WebDriverWait(self.driver, 10)
```

Comment:
I'm working on a change where we'll be able to launch TensorBoard in a separate thread and get a notification, so hopefully we can remove the sleep loop altogether.